libhb/vfr.c

```diff
@@ -538,6 +538,11 @@ static void hb_vfr_close( hb_filter_object_t * filter )
         hb_log("vfr: %d frames output, %d dropped and %d duped for CFR/PFR",
                pv->count_frames, pv->drops, pv->dups );
     }
+    else
+    {
+        hb_log("vfr: %d frames output, %d dropped",
+               pv->count_frames, pv->drops);
+    }
```

Comment:
I notice you log pv->drops, suggesting mode 0 can possibly drop frames (I haven't had tome to check the filter code to see if/when that happens yet)?

I guess what would be critical to check would be a test case where that happens, to make sure the hardware decoders/filter behave correctly in said case.

Reply:
It can happen in https://github.com/HandBrake/HandBrake/blob/ceaca7681556f4a67467dc790ebffe99b1b2dc91/libhb/vfr.c#L637

For example I have a file with crazy timestamps: some frame durations are set to 1/1000000 seconds. HandBrake has got a fixed 90000 timescale, and I guess the pts is rounded and it becomes the same as the previous frame pts, causing the error in #5075 and #5010 .

Reply:
I'll try to make a short sample..

Reply:
Thanks! It would be useful here and probably generally anyway.